agenta-backend/agenta_backend/services/llm_apps_service.py

```diff
@@ -181,7 +181,23 @@ async def batch_invoke(
     list_of_app_outputs: List[InvokationResult] = (
         []
     )  # Outputs after running all batches
-    openapi_parameters = await get_parameters_from_openapi(uri + "/openapi.json")
+    try:
+        openapi_parameters = await get_parameters_from_openapi(uri + "/openapi.json")
+    except Exception as e:
+        logger.traceback.print_exc()
+        for _ in testset_data:
+            list_of_app_outputs.append(
+                InvokationResult(
+                    result=Result(
+                        type="error",
+                        error=Error(
+                            message="An error occurred while invoking the LLM App",
+                            stacktrace=str(e),
+                        ),
+                    )
+                )
+            )
+        return list_of_app_outputs
```

Comment:
What happens in the case where a user has 50 data in their testset, evaluation runs for the first 10, and the eleventh fails?

Are we marking the status of that evaluation as `partial fail`? Or, `fail`? If it's the former, I'd recommend we minus the length of `list_of_app_outputs` from the testset and only record from the failed data point to the last.  If it's the latter (marking the entire evaluation as failed), then we should clear the list_of_app_outputs so we don't end up having duplicate data point results. 

See suggestion:
```python
len_of_app_outputs = len(list_of_app_outputs)
for _ in testsetdata[:len_of_app_outputs]:
    # continue logic
```